client/src/main/java/org/spine3/base/Stringifiers.java

```diff
@@ -206,4 +212,90 @@ protected CommandId doBackward(String s) {
             return result;
         }
     }
+
+    protected static class IntegerStringifier extends Stringifier<Integer> {
+
+        @Override
+        protected String doForward(Integer integer) {
+            return integer.toString();
+        }
+
+        @Override
+        protected Integer doBackward(String s) {
+            try {
+                return Integer.valueOf(s);
+            } catch (NumberFormatException ignored) {
+                final String exMessage = format("Cannot convert %s to Integer.", s);
+                throw new IllegalConversionArgumentException(new ConversionError(exMessage));
+            }
+        }
+    }
+
+    /**
+     * The stringifier for the {@code List} classes.
+     *
+     * <p> The converter for the type of the elements in the list
+     * should be registered in the {@code StringifierRegistry} class
+     * for the correct usage of the {@code ListStringifier} converter.
+     *
+     * @param <T> the type of the elements in the list.
+     */
+    protected static class ListStringifier<T> extends Stringifier<List<T>> {
+
+        private final Class<T> listGenericClass;
+
+        /**
+         * The delimiter for the passed elements in the String representation. Comma by default.
+         */
+        private String delimiter = ",";
```

Comment:
That should be `final` and set to a default value if no custom delimiter is passed.